backend/ops_api/tests/docker-compose.yml

```diff
@@ -31,6 +31,7 @@ services:
       /bin/sh -c "
       . .venv/bin/activate &&
       alembic upgrade head &&
+      DATA=./data_tools/data/ops_event.json5 python ./data_tools/src/import_static_data/import_data.py
```

Comment:
@rajohnson90 I think you'll need an `&&` on the end of this. That is likely why the current pipeline is failing on this